exec-unittests

```diff
@@ -12,17 +12,17 @@ fi
 shift
 
 # The first argument will be which major sub-system to run tests for: the
-# agent side or the server side of the source tree.
+# agent code, the functional test client code, or the server code.
 
 major="${1}"
 shift
-major_list="agent server"
+major_list="agent client server"
 if [[ -n "${major}" ]]; then
-    if [[ "${major}" != "agent" && "${major}" != "server" && "${major}" != "both" ]]; then
-        printf -- "Expected major sub-system to be either 'agent', 'server', or 'both', got '%s'\n" "${major}" >&2
+    if [[ "${major}" != "agent" && "${major}" != "server" && "${major}" != "client" && "${major}" != "all" ]]; then
+        printf -- "Expected major sub-system to be either 'agent', 'server', or 'all', got '%s'\n" "${major}" >&2
```

Comment:
Line 22 should presumably mention `client` (and we should drop "either", since, orthodoxically, that refers to two options and we'll have four).